packages/url_launcher/url_launcher_android/android/src/main/java/io/flutter/plugins/urllauncher/UrlLauncher.java

```diff
@@ -140,10 +144,16 @@ public void closeWebView() {
     return CustomTabsClient.getPackageName(applicationContext, Collections.emptyList()) != null;
   }
 
-  private static boolean openCustomTab(
-      @NonNull Context context, @NonNull Uri uri, @NonNull Bundle headersBundle) {
-    CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder().build();
+  @VisibleForTesting
```

Comment:
Why was this made public for testing? Testing should be done via public interface whenever possible to make tests easier to maintain and less likely to miss important behavior.

Reply:
Reverted this change as well